Sources/XRPLSwift/Xrpl/Utilities/Bip39Mnemonic.swift

```diff
@@ -52,6 +72,93 @@ public final class Bip39Mnemonic {
 
         return PBKDF2SHA512(password: password.bytes, salt: salt.bytes)
     }
+    
+    static func validateMnemonics(_ mnemonics: String, _ language: WordList = .english) throws {
```

Comment:
Making the `validateMnemonics` function as public would enhance its usability and allow for broader application.

Reply:
[82e8b05](https://github.com/girin-app/XRPLSwift/pull/12/commits/82e8b053158561310c0cab4ad53282a1efebc4b1)